core/src/main/res/values/strings.xml

```diff
@@ -68,6 +68,7 @@
   <string name="pref_credits_title">Credits</string>
   <string name="tts_lang_not_supported">The language of this page is not supported. The article may not be properly read.</string>
   <string name="no_reader_application_installed">Could not find an installed application for this type of file</string>
+  <string name="no_email_application_installed">Please install an email service provider or email us at %1s</string>
```

Comment:
"Please install an email client" is everything what the user needs to do... "email service provide" has no sense for most of user and "email us" makes also no sense if there is no email client... Or do I get it wrong?

Reply:
You get it right, I have did the changes.